components/rendering/tests/codeblock_hl_lines.rs

```diff
@@ -40,20 +43,28 @@ fn hl_lines_simple() {
     let permalinks_ctx = HashMap::new();
     let mut config = Config::default();
     config.highlight_code = true;
-    let context = RenderContext::new(&tera_ctx, &config, "", &permalinks_ctx, InsertAnchor::None);
-    let res = render_content(r#"
+    let context =
+        RenderContext::new(&tera_ctx, &config, "", &permalinks_ctx, InsertAnchor::None, EN);
+    let res = render_content(
+        r#"
 ```hl_lines=2
 foo
 bar
 bar
 baz
 ```
-    "#, &context).unwrap();
-    assert_eq!(res.body, colored_html!(
-        @no "foo",
-        @hl "bar",
-        @no "bar\nbaz",
-    ));
+    "#,
+        &context,
+    )
+    .unwrap();
```

Comment:
Rustfmt really messed up this part. Sorry for that. I probably should revert changes to this file.